app/backend/src/couchers/servicers/search.py

```diff
@@ -444,6 +453,22 @@ def UserSearch(self, request, context, session):
             statement = statement.where(
                 User.parking_details.in_([parkingdetails2sql[det] for det in request.parking_details_filter])
             )
+            # limits/default could be handled on the front end as well
+        min_age = request.age_min.value if request.HasField("age_min") else 18
+        max_age = request.age_max.value if request.HasField("age_max") else 200
+
+        statement = statement.where((User.age >= min_age) & (User.age <= max_age))
```

Comment:
It might be better to have these as `if request.HasField():` then only include the `where` in case they are specified. Adding extra `where`s can possibly confuse the query planner?

I don't think it's a big deal!

Reply:
was trying to have a default value assigned as 18 min and 200 max when they are empty.